resources/tests.mjs

```diff
@@ -391,6 +391,30 @@ Suites.push({
     ],
 });
 
+Suites.push({
+    name: "Charts-chartjs",
+    url: "tentative/charts/dist/chartjs.html",
+    async prepare(page) {},
+    tests: [
+        new BenchmarkTestStep("Prepare", (page) => {
```

Comment:
Why isn't this done in prepare function above like other tests?

Reply:
That's because I thought there is value in having this code measured too. The prepare step isn't measured.
In this case there are some iterations, map creation (with everything related to map) and operations, and some trigonometry.
FYI I did the same thing in observable plot, where there are more looping / spreading operations than here, which I thought were valuable to measure too.
Tell me what you think!

Reply:
ok